features/bsm/service/impl/src/main/java/org/opennms/netmgt/bsm/service/internal/edge/ChildEdgeImpl.java

```diff
@@ -70,4 +70,9 @@ public String toString() {
                 .add("child", getChild() == null ? null : getChild().getId())
                 .toString();
     }
+
+    @Override
+    public String getFriendlyName() {
+        return null;
```

Comment:
Can we add a comment, that the friendly name of business services do not have a friendly name.
Otherwise one may think it was forgotten to be implemented?